<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>145</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>150</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>320</width>
    <height>150</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>320</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/window/application_icon_.jpg</normaloff>:/new/prefix1/window/application_icon_.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="background_" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>150</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>320</width>
     <height>150</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QLabel" name="upper_band_">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="close_">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>145</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Тип крепежа</string>
    </property>
    <widget class="QPushButton" name="type_metric_fasteners_">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>15</y>
       <width>135</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Метрический крепеж</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="type_inch_fasteners_">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>39</y>
       <width>135</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Дюймовый крепеж</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>35</y>
      <width>145</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Тип расчета</string>
    </property>
    <widget class="QPushButton" name="selection_fasteners_">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>15</y>
       <width>135</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Подбор крепежа</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="calculatoin_fasteners_">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>39</y>
       <width>135</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Расчет крепежа</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_ok_">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>50</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ОК</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="name_widget_">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки расчета крепежа</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
